### PR TITLE
Added spot requests page

### PR DESCRIPTION
**Pull request by Saksham2k3s**
Spot Request Data

Spot Requests

### CODE REVIEW

File: src/views/PageSpotRequest.vue [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        },
+    },
+};
+</script>
+
+<style scoped>
+.spot-requests-root {
+    padding: 16px;
+    text-align: center;
+    background: #f5f5fb;
```
Comment (arrogant-ant) [inline src/views/PageSpotRequest.vue:172]: please keep the attribute Lexicographically sorted 
Same for all css class.

File: vue.config.js [hunk 2]
```diff
@@ -17,56 +17,56 @@ module.exports = {
...
-                            '/bangalore/parking-near-srirampuram/',
-                            '/bangalore/parking-near-subramanyapura/',
-                            '/bangalore/parking-near-vijayanagar/',
-                            '/bangalore/parking-near-whitefield/',
-                            '/bangalore/parking-near-yelahanka/',
-                            '/bangalore/parking-near-kengeri-gate/',
-                            '/bangalore/parking-near-chickpet/',
-                            '/bangalore/parking-near-yalahanka/',
-                            '/hyderabad/parking-near-hyderabad/',
+                            // '/about/',
```
Comment (arrogant-ant) [inline vue.config.js:20]: dont commit these changes
its just for local testing and development.

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

**Comment by Saksham2k3s** [comment]
After Changes

Spot Requests

**Comment by Saksham2k3s** [comment]
After Changes

Spot Requests

**Pull request closed without merging**